examples/timeline/editing/tooltipOnItemChange.html

```diff
@@ -0,0 +1,81 @@
+<html>
+<head>
+  <title>Timeline | Tooltip on item onUpdateTime Option</title>
+
+  <script src="../../../dist/vis.js"></script>
+  <link href="../../../dist/vis.css" rel="stylesheet" type="text/css" />
+
```

Comment:
Maybe it would be nice to demonstrate also how to overwrite the tooltip style. e.g. with something simple like:
```css
.vis-item .vis-onUpdateTime-tooltip {
    border-radius: 4px;
}
```